engine/gamesys/src/gamesys/components/comp_model.cpp

```diff
@@ -413,7 +486,14 @@ namespace dmGameSystem
             }
 
             ro->m_Textures[i] = texture_res ? texture_res->m_Texture : 0;
+
+            if (ro->m_Textures[i] == 0 && !first_free_index_set)
+            {
+                first_free_index_set = true;
+                first_free_index = i;
+            }
         }
+        return first_free_index;
```

Comment:
We bind the bone cache texture to the first free index we can find. It's an internal resource (like the font cache), and as such we shouldn't expose it to the materials to the users.